jyulb/__init__.py

```diff
@@ -1,5 +1,5 @@
-from simphony.engine import ABCEngineExtension
-from simphony.engine import EngineInterface
+from simphony.extension import ABCEngineExtension
+from simphony.extension import EngineInterface
```

Comment:
@tuopuu Sorry for the inconvenience, there was a cyclic problem and I had to remove `extension` package. Would you mind  importing from `simphony.engine` again?